src/legacy/core_plugins/console/np_ready/public/application/components/console_menu.tsx

```diff
@@ -91,6 +103,41 @@ export class ConsoleMenu extends Component<Props, State> {
     this.props.autoIndent(event);
   };
 
+  getCopyToast = () => {
+    const copyToast = [
+      {
+        title: i18n.translate('console.copyRequestAsCurl', {
+          defaultMessage: 'Request copied as cURL',
+        }),
+        color: 'success',
+      },
+    ];
+
+    const actualToastId = this.state.toastId;
+    this.setState({ toastId: this.state.toastId + 1 });
+
+    return {
+      id: actualToastId,
+      ...copyToast[0],
+    };
+  };
+
+  addCopyToast = () => {
```

Comment:
Would it be possible to combine `getCopyToast` and `addCopyToast` into one function? Something like:

```ts
addCopyToast = () => {
  const toast = {
      id: this.state.toastId,
      title: i18n.translate('console.copyRequestAsCurl', {
        defaultMessage: 'Request copied as cURL',
       }),
      color: 'success',
    };

    this.setState({
      toastId: this.state.toastId + 1,
      toasts: this.state.toasts.concat(toast),
    });
}

Reply:
Ok, done.